README.md

```diff
@@ -16,52 +17,29 @@ which uses the [ansible playbook](https://github.com/RedHatOfficial/ocp4-helpern
 
 This project also leverages the same ansible playbook internally for OCP deployment on PowerVM LPARs managed via PowerVC.
 
-Run this code from either Mac or Linux (Intel) system.
 
-:heavy_exclamation_mark: *This automation is intended for test/development purposes only and there is no formal support. For bugs/enhancement requests etc. please open a GitHub issue*
+:heavy_exclamation_mark: *For bugs/enhancement requests etc. please open a GitHub issue*
 
-# Pre-requisites
+:information_source: **This (main) branch must be used with latest OCP pre-release versions only. For stable releases please checkout specific release branches - {release-4.5, release-4.6 ...} and follow the docs in the specific branches.**
 
-You need to identify a remote client machine for running the automation. This could be your laptop or a VM.
+## Automation Host Prerequisites
 
-### Operating Systems
+The automation needs to run from a system with internet access. This could be your laptop or a VM with public internet connectivity. This automation code have been tested on the following 64-bit Operating Systems:
+- Mac OSX (Darwin)
+- Linux (x86_64)
+- Windows 10
```

Comment:
We do not have windows support for PowerVC yet. It is still pending in the issues list.

Reply:
Ok.. I'll remove it